gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Thu Dec 19 23:35:33 CET 2019
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
```

Comment:
Jaki jest powód zmiany bin na all?

Reply:
Nie wiem czemu, ale to zmienia mi na siłę...